test/glific/wa_managed_phones_test.exs

```diff
@@ -163,4 +169,84 @@ defmodule Glific.WAManagedPhonesTest do
                WAManagedPhones.fetch_wa_managed_phones(attrs.organization_id)
     end
   end
+
+  test "status/2 webhook should update the status on wa_managed_phone", %{
+    staff: user,
+    conn: conn
+  } do
+    wa_managed_phone = Fixtures.wa_managed_phone_fixture(%{organization_id: user.organization_id})
+
+    params = %{
+      "phoneId" => wa_managed_phone.phone_id,
+      "phone_id" => wa_managed_phone.phone_id,
+      "pid" => "a3ff8460-c710-11ee-a8e7-5fbaaf152c1d",
+      "product_id" => "a3ff8460-c710-11ee-a8e7-5fbaaf152c1d",
+      "status" => "active",
+      "type" => "status"
+    }
+
+    conn = StatusController.status(conn, params)
+
+    assert conn.status == 200
+
+    # small delay to allow background task to complete
+    :timer.sleep(100)
```

Comment:
If we individually test the function inside the task somehwere, then we don't have to wait here to test the result.